src/PhpSpreadsheet/Reader/Xls.php

```diff
@@ -439,6 +439,14 @@ public function canRead($pFilename)
         }
     }
 
+    /**
+     * @param string $codepage
+     */
+    public function setCodepage($codepage): void
```

Comment:
You might want to consider typehinting, and some form of validation here

Reply:
> You might want to consider typehinting, and some form of validation here

done